WcaOnRails/spec/requests/results_submission_spec.rb

```diff
@@ -54,55 +54,36 @@
     end
 
     describe "Posting results" do
+      let(:results_submission) do
```

Comment:
Nit: this name implies an instance of `ResultsSubmission`, maybe it could be `results_submission_params` instead?